paddle/framework/data_layout.h

```diff
@@ -33,5 +36,23 @@ inline DataLayout StringToDataLayout(const std::string& str) {
   }
 }
 
+inline std::string DataLayoutToString(const DataLayout& data_layout) {
+  switch (data_layout) {
+    case kNHWC:
+      return "NHWC";
+    case kNCHW:
+      return "NCHW";
+    case kAnyLayout:
+      return "ANY_LAYOUT";
+    default:
```

Comment:
'NCDHW' should also been taken into account.

Reply:
yes, all the format will be added later~